source/usage-examples/count.txt

```diff
@@ -0,0 +1,61 @@
+=============================================
+Count the Number of Documents in a Collection
+=============================================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+The Node.js driver provides two methods for counting the number of
+documents in a collection:
```

Comment:
While "the number of documents in a collection" isn't inaccurate, I feel like "counting documents in a collection" better conveys the idea that these methods have different counting use cases, since "the number of documents in a collection" is very similar to the wording used to explain ``estimatedDocumentCount()``

Reply:
Done.